pkg/operation/botanist/component/kubeapiserver/deployment.go

```diff
@@ -634,7 +634,7 @@ func (k *kubeAPIServer) handleSNISettings(deployment *appsv1.Deployment) {
 }
 
 func (k *kubeAPIServer) handleLifecycleSettings(deployment *appsv1.Deployment) {
-	if !versionutils.ConstraintK8sGreaterEqual116.Check(k.values.Version) {
+	if versionutils.ConstraintK8sLess116.Check(k.values.Version) {
```

Comment:
I thought just to remove the negation and switch the `if` and the `else` block but this is also fine :D